niworkflows/tests/test_viz.py

```diff
@@ -149,3 +153,25 @@ def test_compcor_variance_plot(tmp_path):
     out_file = str(out_dir / "variance_plot_short.svg")
     metadata_file = os.path.join(datadir, "confounds_metadata_short_test.tsv")
     viz.plots.compcor_variance_plot([metadata_file], output_file=out_file)
+
+
+@pytest.fixture
+def create_surface_dtseries():
+    """Create a dense timeseries CIFTI-2 file with only cortex structures"""
+    out_file = _create_dtseries_cifti(
+        timepoints=10,
+        models=[
+            ('CIFTI_STRUCTURE_CORTEX_LEFT', np.random.rand(29696, 10)),
+            ('CIFTI_STRUCTURE_CORTEX_RIGHT', np.random.rand(29716, 10)),
+        ],
+    )
+    yield str(out_file)
```

Comment:
If you add a `print` here, I'm pretty sure you'll get something in the current working directory, and not in `tmp_path`.

Reply:
nope, I checked this - `/private/var/folders/r6/mcdps6rs37x2ttd1fn032yy40000gp/T/tmpdmdmvolz/test.dtseries.nii`

Reply:
Huh. I guess I really don't understand how pytest works then.